### ISSUE TITLE
update Perch model to version 8

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/bioacoustics-model-zoo#12]
https://www.kaggle.com/models/google/bird-vocalization-classifier/tensorFlow2/bird-vocalization-classifier/8

- enables batching
- returns logits for intermediate outputs, and returns spectrogram

So, will have to refactor the code slightly to (1) enable batching and (2) handle and optionally return the additional outputs

**Comment by sammlapp** [comment]
resolved in release #13
